_data/toc.yml

```diff
@@ -1,47 +1,41 @@
 - title: Geyser Documentation
   url: "geyser/"
   links:
-    - title: "Setup"
+    - title: Setting up Geyser
       url: "geyser/setup/"
       children:
         - title: Plugin Setup
           url: "geyser/setup/#plugin-setup"
-        - title: Videos
-          url: "geyser/setup/#setup-videos"
-    - title: "Standalone"
+        - title: Supported Hosting Providers
+          url: "geyser/supported-hosting-providers/"
+        - title: Using Geyser with Consoles
+          url: "geyser/using-geyser-with-consoles/"
+        - title: Using playit.gg with Geyser
+          url: "geyser/playit-gg/"
+    - title: Setting up Geyser-Standalone
       url: "geyser/standalone/"
       children:
-        - title: Termux (Android)
-          url: "geyser/standalone/#termux-android"
-        - title: NewTerm 2 (iOS)
-          url: "geyser/standalone/#newterm-2-ios"
-    - title: FAQ
-      url: "geyser/faq/"
+        - title: Creating a Startup Script
+          url: "geyser/creating-a-startup-script/"
```

Comment:
Maybe have this as a 'setting up Geyser' section as well, just like playit.gg is on there